release-notes/release-notes59.md

```diff
@@ -53,6 +53,7 @@ For information on:
   - `mceEmoticons` opens the Emoticons dialog.
   - `mceWordCount` displays the Word Count summary dialog.
   - `mceTemplate` shows the Template dialog.
+- New [`iframe_aria_text`]({{site.baseurl}}/configure/accessibility/#iframe_aria_text) option to allow setting the editor iframe title attribute.
```

Comment:
```suggestion
- A new `iframe_aria_text` option for customizing the `title` attribute on the editor iframe. For details, see: [Accessibility options - `iframe_aria_text`]({{site.baseurl}}/configure/accessibility/#iframe_aria_text).
```

Reply:
Ensure links are at the end of sentences where possible for readability and accessibility. https://www.tiny.cloud/docs/configure/contributing-docs/#linksandcross-references